content/posts/newsletter/2020-03-15.md

```diff
@@ -10,11 +10,14 @@ category: Newsletter
 
 ## Intro
 
-Welcome to the XMPP newsletter covering the month of February 2020. We are very sorry for the late publish. :(
-Anyhow, this is a big one!
+Welcome to the XMPP newsletter covering the month of February and March 2020.
+
+We had a big delay releasing and finalizing the February release of the XMPP Newsletter. So the decision was made to combine both into this version - apologies! 
```

Comment:
Personally, I wouldn't write this. I don't think it's necessary and writing "big delay" reflects poorly on us. I would just not mention it at all. My personal opinion of course.